account_banking_mandate/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=======================
+Account Banking Mandate
+=======================
+
+This module adds a generic model for banking mandates.
+These mandates can be specialized to fit any banking mandates (such as sepa or lsv).
+
+A banking mandate is attached to a bank account and represents an
+authorization that the bank account owner gives to a company for a
+specific operation (such as direct debit).
+You can setup mandates from the accounting menu or directly from a bank
+account.
+
+Installation
+============
+
+This module depends on :
+* account_payment
+
+This module is part of the OCA/bank-payment suite.
+
+Configuration
+=============
+
+TODO
+
+Usage
+=====
+
+To use this module, see menu "Accounting > payment > SEPA direct debit mandates" 
```

Comment:
This does not match with the views. I think the menu would be 'Invoicing > Payments > Banking Mandates' instead.